presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoDatabaseMetaData.java

```diff
@@ -264,7 +267,7 @@ public String getTimeDateFunctions()
     public String getSearchStringEscape()
             throws SQLException
     {
-        return "\\";
+        return String.valueOf(SEARCH_ESCAPE);
```

Comment:
Make this a constant as well, so we always return a constant string (seems better than creating a new string each time)